playbooks/provisioning/openstack/sample-inventory/group_vars/OSEv3.yml

```diff
@@ -5,7 +5,7 @@ openshift_master_default_subdomain: "apps.{{ env_id }}.{{ public_dns_domain }}"
 
 # NOTE(shadower): do not remove this line, otherwise the default node labels
 # won't be set up.
-openshift_node_labels: "{{ openstack.metadata.node_labels }}"
+openshift_node_labels: "{{ metadata.node_labels }}"
```

Comment:
My openshift installation failed with `{{ metadata.node_labels }}: 'metadata' is undefined`.

We're not setting `metadata` anywhere and openshift-ansible/.../byo uses the static inventory so it doesn't see the shade output.

I think we need to set metadata in the generated `hosts` file. Or maybe even just set the node labels directly in the inventory to each group.